server/test/durablejobs/DurableJobRunnerTest.java

```diff
@@ -140,21 +142,30 @@ public void runJobs_runsJobsThatAreReady() {
   }
 
   @Test
-  public void runJobs_jobNotFound() {
+  public void runJobs_jobNotFound_deletesJobFromDb() {
```

Comment:
I got turned around trying to mock out the call to `delete` to test the other code path (when delete fails). If you think we need a test for that maybe we can pair on it.